src/objects/setup_configuration/models/objecttypes.py

```diff
@@ -0,0 +1,18 @@
+from django_setup_configuration.fields import DjangoModelRef
+from django_setup_configuration.models import ConfigurationModel
+from pydantic import Field
+from zgw_consumers.models import Service
+
+from objects.core.models import ObjectType
+
+
+class ObjectTypeConfigurationModel(ConfigurationModel):
```

Comment:
@swrichards @joeribekker I don't think this was very clear from the issue, but I think the scope of setup-configuration here was to be able to configure the connection from Objects API to Objecttypes API, and not to be able to configure objecttypes, right?

Reply:
Objecttypes can be created with the API once a token is obtained. So, no, objecttypes should not be created via the YAML file.